assets/stylesheets/custom.css.scss

```diff
@@ -28,3 +38,30 @@ body {
 .panel-body {
   font-size: 1.33em;
 }
+
+.text-center h1 {
+  font-size: 4.5em;
+  margin: 55px 0;
+}
+
+.btn.btn-primary.btn-lg.show-report {
+  padding: 9px;
+  margin-top: 68px;
+}
+.btn.btn-primary.btn-lg {
+  padding: 10px;
+  font-size: 2.2em;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  margin-right: -10px;
+}
```

Comment:
Don't want to overwrite Bootstrap's stuff. Change specific instances rather than the whole Bootstrap behavior.